### ISSUE TITLE
`lazygit` is not available on `apt-get`

### ISSUE THREAD

**Issue by Qazalbash** [EliTheSurfer/EliVim-VSCode#5]

**Comment by EliTheSurfer** [comment]
Hello @Qazalbash , does this (pull request)[https://github.com/EliTheSurfer/EliVim-VSCode/pull/6] fix your issue?

**Comment by Qazalbash** [comment]
@EliTheSurfer Yes, it does. Thanks for such a quick response. I have one more request irrelevant to this issue. I installed vs code using snap and there is no directory `/usr/share/code` so I can not run this command `sudo chown -R $(whoami) /usr/share/code` to allow vs code to change itself. Do you have any heads over this problem?

**Comment by EliTheSurfer** [comment]
@Qazalbash unfortunately no since I dont have any ubuntu machine. Feel free to update the readme section if you find any solution 👍
